lib/screens/signin_screen.dart

```diff
@@ -74,13 +72,7 @@ class _LoginPageState extends State<LoginPage> {
     try {
       await _emailAuth.signIn(email, password);
 
-      Navigator.pushAndRemoveUntil(
-        context,
-        MaterialPageRoute(
-          builder: (context) => Home(),
-        ),
-        (Route<void> route) => false,
-      );
+      Navigator.of(context).pushNamed('/home');
```

Comment:
Is it possible to keep `(Route<void> route) => false,` as we don't want the user to go back to sign-in page with the back button?